test/e2e/util/util.js

```diff
@@ -13,6 +13,12 @@
 /* global module, browser */
 var util = module.exports = {};
 
+var EC = protractor.ExpectedConditions;
+
 util.loadTestPage = function (pageName) {
   browser.get('http://localhost:3000/' + pageName + '.html');
 };
+
+util.waitForElement = function (el) {
+  browser.wait(EC.presenceOf(el), 1000);
```

Comment:
Can we do 'wait for something to be visible' rather fighting with time.
(I thought I added at first place but apparently I forgot ...)